lib/index.js

```diff
@@ -265,6 +264,18 @@ function normalizeFunctionSignature(signature, callback) {
  */
 
 module.exports.render = function(opts, cb) {
+  // Add include paths from SASS_PATH environment variable, if present
```

Comment:
This normalization probably belongs up in the getOptions `options.includePaths = (options.includePaths || []).join(path.delimiter);`, then there is no change anything else